crates/ra_project_model/src/json_project.rs

```diff
@@ -20,8 +20,17 @@ pub struct Crate {
     pub(crate) root_module: PathBuf,
     pub(crate) edition: Edition,
     pub(crate) deps: Vec<Dep>,
+
+    // This is the preferred method of providing cfg options.
+    #[serde(default)]
+    pub(crate) cfg: FxHashSet<String>,
```

Comment:
cfgs please :)

Reply:
@matklad had used `cfg` in https://github.com/rust-analyzer/rust-analyzer/issues/4310#issuecomment-623970585, so I was following that.  I also have a slight preference for `cfgs`.

The fields on `CfgOptions` that this is converted to are both plural (`atoms`, `key_values`).

Reply:
Don't have strong opinion, but like cfg slightly more

Reply:
I defer to you @matklad :)